charts/kafka-operator/templates/crds.yaml

```diff
@@ -3493,6 +3493,18 @@ spec:
                     type: object
                   type: array
               type: object
+            disruptionBudget:
+              description: DisruptionBudget defines the configuration for PodDisruptionBudget
+              properties:
+                budget:
+                  description: The budget to set for the PDB, can either be static
+                    number or a percentage
+                  pattern: '[0-9]+(\.[0-9]+)?(%)?'
```

Comment:
How about this regex: `^[0-9]+$|^[0-9]{1,2}%$|^100%$` to not allow invalid percentage values such as 1000%

Reply:
All for not allowing invalid percentage, do we want to invalidate "33.33%" though?

Reply:
Do you see any use case when fractional percentage would be useful?

Reply:
How about
`^[0-9]+$|^[0-9]{1,2}(\.[0-9]+%)?$|^100%$` ?

Reply:
You might be right, in terms of use cases, fractional percentages would only be helpful here when going over 1000, otherwise they are rendered useless by the floor function

Reply:
Let's go without fractional and change it if we need to.

Reply:
that's what I was thinking as well. The fractional percentage would have a noticeable impact only in case of broker count over 1000, even than compared to the total number of brokers the diff would negligible